<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesSettingsWidget</class>
 <widget class="QWidget" name="FilesSettingsWidget">
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbPhotos">
     <property name="title">
      <string>Photos</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QListView" name="lvPhotos">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QToolButton" name="tbAdd">
          <property name="toolTip">
           <string>Add a photo</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/images.qrc">
            <normaloff>:/images/files_add</normaloff>:/images/files_add</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbFolderAdd">
          <property name="toolTip">
           <string>Add photos in folder</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/images.qrc">
            <normaloff>:/images/folder_photo</normaloff>:/images/folder_photo</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbDel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete photo</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/images.qrc">
            <normaloff>:/images/files_delete</normaloff>:/images/files_delete</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tbMoveUp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Move photo up</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/images.qrc">
            <normaloff>:/images/arrow_up</normaloff>:/images/arrow_up</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbMoveDown">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Move photo down</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/images.qrc">
            <normaloff>:/images/arrow_down</normaloff>:/images/arrow_down</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlPreview">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections/>
</ui>
